include/arinc429/arinc429.h

```diff
@@ -327,23 +340,27 @@ namespace eld
             }
 
             template<typename T>
-            void set_integral_value(const T & /*value*/,
-                                    traits::word_raw_type & /*wordRaw*/,
-                                    size_t /*lsb*/,
-                                    size_t /*msb*/,
+            void set_integral_value(const T & value,
+                                    traits::word_raw_type & wordRaw,
+                                    size_t lsb,
+                                    size_t msb,
                                     std::false_type /*is_signed*/)
             {
-                // TODO: implement
+                size_t unused_bit_count = 32 - (msb - lsb + 1);
+                uint32_t clamped_value = (uint32_t(value) << unused_bit_count) >> unused_bit_count;
+                clamped_value = clamped_value << (lsb - 1);
+                wordRaw = wordRaw | clamped_value;
```

Comment:
need to erase bits in assignable range. `wordRaw = (wordRaw & eraseBits) | clamped_value;`